plugins/catalog-react/src/components/EntityPeekAheadPopover/CardActionComponents/EmailCardAction.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Button, Tooltip } from '@material-ui/core';
+import EmailIcon from '@material-ui/icons/Email';
+import React from 'react';
+
+/**
+ * Email Card action link
+ *
+ * @private
+ */
+export const EmailCardAction = ({ email }: { email: string }) => {
+  return (
+    <Tooltip title={`Email ${email}`}>
+      <Button target="_blank" href={`mailto:${email}`} size="small">
```

Comment:
To get it behaviorally the same as our links, could you try out whether `<IconButton component={Link} to={<url>} size="small">` works as intended? I think it might. See for example https://github.com/backstage/backstage/blob/e253423ae4bea08357f49e072d6d46e4ff56b433/plugins/catalog/src/components/AboutCard/AboutCard.tsx#L172

Using our Link of course, not the one in MUI. Adding aria-label is bonus points

Reply:
done